packages/quantic/cypress/integration/document-suggestion/document.suggestion.expectations.ts

```diff
@@ -0,0 +1,59 @@
+import {InterceptAliases} from '../../page-objects/case-assist';
+import {should} from '../common-selectors';
+import {
+  DocumentSuggestionSelector,
+  DocumentSuggestionSelectors,
+} from './document-suggestion-selectors';
+
+function documentSuggestionExpectations(selector: DocumentSuggestionSelector) {
+  return {
+    displayAccordion: (display: boolean) => {
+      selector
+        .accordion()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the accordion`);
+    },
+
+    displayLoading: (display: boolean) => {
+      selector
+        .loadingSpinner()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the loading spinner`);
+    },
+
+    numberOfSuggestions: (value: number) => {
+      selector
+        .accordionSections()
+        .should('have.length', value)
+        .logDetail(`should display ${value} document suggestions`);
+    },
+
+    logClickingSuggestion: (index: number) => {
+      cy.wait(InterceptAliases.UA.SuggestionClick)
+        .then((interception) => {
+          const analyticsBody = interception.request.body;
+          selector
+            .accordionSection(index)
+            .invoke('attr', 'data-id')
+            .should('eq', analyticsBody.svc_action_data.suggestionId);
```

Comment:
is there anything else that we will log for UA and should be validated? like actionCause, or something else?

Reply:
Actually nope, we have only the suggestions id and the actionCause is in the name of the event so that's all what we are logging for UA.

Reply:
yes there is some additional information about the suggestion like the title and the URI and the position of this suggestion. Just added some tests to make sure we are sending everything correctly.